MembraneRTC/Sources/MembraneRTC/MembraneRTC.swift

```diff
@@ -160,6 +163,12 @@ public class MembraneRTC: MulticastDelegate<MembraneRTCDelegate>, ObservableObje
      */
     public func createVideoTrack(videoParameters: VideoParameters, metadata: Metadata) -> LocalVideoTrack {
         let videoTrack = LocalVideoTrack.create(for: .camera, videoParameters: videoParameters)
+        
+        if state == .connected {
+            connection?.add(videoTrack.rtcTrack(), streamIds: [localStreamId])
```

Comment:
is it ok to add a track twice?

Reply:
I think that this is for adding a track after creating a peerConnection

Reply:
@graszka22 
It shouldn't be adding track twice because of the:
```Swift
if state == .connected
```
statement.

Correct me if I missed something, but adding tracks to the connection before this change only worked if the client would add all of them before connecting (like you mentioned [here](https://github.com/membraneframework/membrane-webrtc-ios/pull/2/files#r928818967)). The code was gathering all tracks into `localTracks` bucket and then adding all of them when connecting. After that it was impossible to add another track because the code would still append `localTracks`, but the actual `connection.add(...)` would never be called because the function that does that (`setupPeerConnection()`) is only called once - when the connection is being made. This change fixes that problem.

Reply:
ok, my bad, it's ok 👍